### PR TITLE
Tokens records search.

### PR DESCRIPTION
**Pull request by benoitdevos**
* Add a new button to check if a Tokens record with id is present in the collection.
* If the input value is not a hash, it will be considered as a business key, and hashed before search.
* A maximum size of 1000 is set to prevent crashing the front in case of huge tokens records set - to be adapted when real backend paging is available.
* Some minor fixes.

logion-network/logion-internal#1158

### CODE REVIEW

File: src/loc/record/TokensRecordFrame.tsx [hunk 1]
```diff
@@ -65,3 +155,52 @@ export default function TokensRecordFrame(props: { contributionMode?: Contributi
...
+                </>
+            )
+        case "NEGATIVE":
+            return (
+                <Row className="CheckResultFeedback result-negative" id={ `feedback-${ state }` }>
+                    <Col>
+                        <p>
+                            Check result: <span className="label-negative">negative</span><br />
+                            The Tokens Record - defined by the ID you submitted - has no match and is NOT part of
+                            the current<br />
```
Comment (gdethier) [inline src/loc/record/TokensRecordFrame.tsx:191]: Why a line break here?

**Review by gdethier — APPROVED** [review]
Good job! I have a small question and a general remark: the limit of 1000 elements should be tested. Indeed, we will very soon have a project with a little more than 15000 records. If is still works, even slowly, with that number of records, we do not need to implement backend-side paging. If it does not, then we have to address this urgently.

File: src/loc/record/TokensRecordFrame.tsx (hunk 1 above)
Comment (benoitdevos) [reply src/loc/record/TokensRecordFrame.tsx:191]: It's just easier to read (vs a one-line paragraph). And also it's consistent with https://github.com/logion-network/logion-wallet/blob/117b598661a40af8e202ea9d03508936d44f6b3f/src/loc/CollectionLocItemChecker.tsx#L337-L342

**Comment by benoitdevos** [comment]
> Good job! I have a small question and a general remark: the limit of 1000 elements should be tested. Indeed, we will very soon have a project with a little more than 15000 records. If is still works, even slowly, with that number of records, we do not need to implement backend-side paging. If it does not, then we have to address this urgently.

As discussed, actual might not be necessary. If yes, a proper bacend-side paging should then be planned in an upcoming sprint.

**Pull request merged**